src/defaults.mligo

```diff
@@ -57,8 +67,18 @@ let default_storage (data, config_data : initial_storage_data * initial_config_d
       ]
     );
     frozen_token_id = frozen_token_id;
-    start_time = data.now_val
-}
+    start_time = data.now_val;
+    quorum_threshold_at_stage =
+      { last_updated_period = 0n
+      ; quorum_threshold = to_unsigned(quorum_threshold)
+      };
+    period_staked =
+      { last_updated_period = 0n
+          // Staked tokens are tracked for a pair of proposal/voting periods.
+          // So should only contain period no of a proposing period.
```

Comment:
The comment seems to contradict the value, since `0n` is a `voting` stage.

However I think the comment is incorrect, while it is true that we want to track only per-cycle (proposal + voting stage) we probably want to allow for both odd and even number to be stored here, just for redundancy (e.g. what happens if nobody calls any entrypoint in a `proposing` stage?)

Reply:
Have changed it to contain `1n` by default so that the inital quorum threshold won't get overwritten in cycle 1, when the first proposal is raised.